ui/js/selectors/app.js

```diff
@@ -39,34 +39,37 @@ export const selectPageTitle = createSelector(
   (page, params) => {
     switch (page) {
       case 'search':
-        return params.query ? `Search results for ${params.query}` : 'Search'
+        return params.query ? __("Search results for %s", params.query) : __('Search')
       case 'settings':
-        return 'Settings'
+        return __('Settings')
       case 'help':
-        return 'Help'
+        return __('Help')
       case 'report':
-        return 'Report'
+        return __('Report')
       case 'wallet':
+        return __("Wallet")
       case 'send':
+        return __("Send")
       case 'receive':
+        return __("Receive")
       case 'rewards':
-        return page.charAt(0).toUpperCase() + page.slice(1)
```

Comment:
I don't know if it helps, but wrapping __() around the output, even if it's not a build-in String (e.g. rewards which are loaded from the server) also work with y18n, you just need to run the app once and then it's added to the language file automatically